index.html

```diff
@@ -70,12 +65,7 @@
 
       <div class="panel radius text-center">
         <p><em>Bycatch</em> is now available for <strong>$14</strong> plus shipping and tax.<br>We accept <strong>PayPal</strong> or <strong>Stripe</strong> and offer <strong>international shipping</strong>.<br>Supply is limited, so order your copy now!</p>
-        <a href="#" class="button radius snipcart-add-item"
-        data-item-id="4"
-        data-item-name="Bycatch card game"
-        data-item-price="14.00"
-        data-item-weight="163"
-        data-item-url="http://bycat.ch"><i class="fa fa-shopping-cart fa-lg"></i> Order</a>
+        <a href="mailto:bycatch@hubbub.eu?subject=Bycatch order" class="button radius"><i class="fa fa-shopping-cart fa-lg"></i>  Email to order</a>
```

Comment:
How would they know what to e-mail us? Or they send us an e-mail and we have a template response for it to figure out how many copies and how to pay us and stuff?